### DISCUSSION TITLE
New Fluent Bit Sandbox!

### DISCUSSION THREAD

**Discussion by agup006** [fluent/fluent-bit#5233] [Announcements]
We have a new Fluent Bit sandbox that is available here: https://play.instruqt.com/Fluent/tracks/fluent-bit-getting-started-101

We will continually evolve this to add more content based on what the community wants. Be sure to add commends / feedback below!

**Comment by l2dy** [comment]
It's not clear that changes in "Fluent Bit Configuration" tab need to be saved manually by pressing an icon, and I did not find a keyboard shortcut for that.

**Reply by agup006** [reply]
Ah great feedback, I'll add a note that users should press that for saving

**Comment by seveas** [comment]
The main reason for me to play with fluent on the command line is to ~~mess about with~~ troubleshoot filters and parsers, so some examples of how to do that would be nice.

**Comment by seveas** [comment]
A common question is: How do I route my logs to different (splunk indexes|elasticsearch indexes|kafka topics...) per (systemd service|kubernetes container...)

**Comment by patrick-stephens** [comment]
Looks like we have Windows VM support from Instruqt now as well so @seveas that'll help with my similar need to test things on platforms I do not easily have access too!

**Comment by brunson** [comment]
I get a 404 when accessing the sandbox now. I am logged into instruqt

**Reply by patrick-stephens** [reply]
There was a change on the Instruqt side to include a token now in the URL. Let me see if I can find the right link on mobile.

https://play.instruqt.com/embed/Fluent/tracks/fluent-bit-getting-started-101?token=em_S2zOzhhDQepM0vDS

Via the blog post: https://calyptia.com/2022/04/12/introducing-the-calyptia-sandbox-environment-for-fluent-bit/

Seems to work for me.

@agup006 can you update your original post?

**Reply by brunson** [reply]
Works now, thanks!

**Comment by haitam-boukaa** [comment]
Hitting the start button gives the following error:

![image](https://github.com/fluent/fluent-bit/assets/147826900/21f423b1-5beb-41dd-b4e5-fdaa07a38d03)

**Reply by patrick-stephens** [reply]
There are various limits/protections in place so I would retry later - we've seen issues with commercial networks looking like they're sharing an IP so it appears to be a DOS attack when lots of people in the org try to start a track.
